static/css/wfrp4e.css

```diff
@@ -14760,23 +14760,21 @@ textarea,
   background: #352516;
 }
 
-.active-effect-sheet .ace-editor {
-  height: 300px;
+.script-config div:has( > .ace-editor) {
+  height: calc(100% - 40px);  
+  display: flex;
+  flex-direction: column;
 }
 
-.active-effect-sheet .ace-editor > *{
-  height: 300px;
+.script-config .ace-editor {
+  height: 100% !important;
+  flex: 1 1 auto;
 }
 
-.active-effect-sheet textarea {
+.script-config textarea {
```

Comment:
The real pain I was experiencing with the script window is making sure it works with both normal scripts and dialog scripts (which have 3 other subscript fields).

Reply:
I will play around with that.

Reply:
Testing it, changing to a dialog effect simply disables the ace editor, which I think is fine. Feel free to tinker but comment when you're satisfied and want me to merge.